Translated techlab 3.5 to english (#31)
appuio/techlab: additional-labs/logging_efk_stack.md

# Lab: Logging EFK Stack

> Witch OpenShift an EFK (Elasticsearch, Fluentd, Kibana) Stack will be provided. It collects all the log files, rotates and aggregates them. Kibana allows it to search through the logs and to filter as well as to represent them graphically.

Reviewer: ```suggestion
OpenShift provides an integrated EFK (Elasticsearch, Fluentd, Kibana) stack used to collect, rotate and aggregate all log files. Kibana is then used for visualizing, searching and filtering these logs.
```
